running-recipes/multi-module-maven.md

```diff
@@ -0,0 +1,28 @@
+# Running OpenRewrite on a multi-module Maven project
+
+If you have a Maven project with multiple modules, you may run into errors finding and running the [OpenRewrite Maven plugin](https://github.com/openrewrite/rewrite-maven-plugin).
+
+There is currently an [open issue](https://github.com/openrewrite/rewrite-maven-plugin/issues/475) to better support multi-module Maven projects that you can +1 on if you'd like. 
+
+In the meantime, one way to work around this is to add a profile on the parent POM:
+
+```xml
+<profile>
+  <id>openrewrite</id>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.openrewrite.maven</groupId>
+        <artifactId>rewrite-maven-plugin</artifactId>
+        <version>5.2.6</version>
+        <configuration>
+          <activeRecipes>
+            <recipe>org.openrewrite.java.cleanup.Cleanup</recipe>
+          </activeRecipes>
+          <failOnDryRunResults>true</failOnDryRunResults>
+        </configuration>
+      </plugin>
+    </plugins>
+  </build>
+</profile>
+```
```

Comment:
I'd add here how people can activate the profile and run the recipes for those less familiar with Maven profiles. Something like

> You can then run OpenRewrite from any submodule using `mvn -Popenrewrite rewrite:run`.

Haven't tested that, but I think it ought to work.

Reply:
Updated. Gonna merge this in for now as it's past your end of day - if my changes are wrong or you think I should make additional updates please let me know. Happy to make more changes.